packages/code-du-travail-nlp/api/sem_search.py

```diff
@@ -0,0 +1,146 @@
+import json
+import tensorflow as tf
+import tensorflow_hub as hub
+import numpy as np
+import tf_sentencepiece
+import unidecode
```

Comment:
`unidecode` n'est pas un package standard et est environ 9x plus lent que la solution avec un package standard Python, `unicodedata`
```suggestion
import unicodedata
```